src/client/src/App/Trades/TradesGrid/utils.ts

```diff
@@ -0,0 +1,9 @@
+//Using because react-window seems to warp flex values in its items
+export const getWidthPercentage = (widths: number[], width: number) =>
+  width / (widths.reduce((acc, curr) => acc + curr) / 100)
+
+export function convertRemToPixels(rem: number) {
+  return process.env.NODE_ENV !== "test"
```

Comment:
as disc, not happy with test code in main code .. suggested move test impl to module in __mocks__